server/database/seeding.py

```diff
@@ -0,0 +1,6 @@
+from database.models import db, User
+
+
+def seed():
+    User(email='test@test.com', password='testPW').save()
+    User(email='admin@admin.com', password='adminPW').save()
```

Comment:
Could we put a check to see if objects with these emails already exist? Just for the purposes of not having duplicates. Another option could be to potentially add the `unique=true` parameter to the User model and surround these in a try-catch block.

Reply:
With the current seeding solution, it seems to be double adding these documents to the DB. My intention was to set the unique parameter, but with it double adding it obviously ran into some issues. Left a comment on the User model and am looking into better seeding solutions

Reply:
Update: Figured out seeding through javascript scripts that work as intended. Check out the changes on the README